src/Engine/ProtoCore/Lang/CallSite.cs

```diff
@@ -366,8 +395,197 @@ public void UpdateCallSite(int classScope, string methodName)
             this.methodName = methodName;
         }
 
-        #region Support Methods
```

Comment:
The following chunk is just moving methods around to group them in the correct regions. We had public methods in the `support methods` region, and private methods in the `constructors` region. It was a mess.